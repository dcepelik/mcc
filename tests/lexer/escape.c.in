"String \awith\b various\t\"escape\"\n\nsequences\x20within\u0020\U00000020it\?"

"single line string con\
tained in multiple physical lines"

L"string"
U"string"
u"string"
u8"string"

"an unterminated string

'\x20'
'\u0020'
'\U0020'
'u
'\\'
