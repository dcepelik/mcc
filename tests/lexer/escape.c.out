"String \awith\b various\t\"escape\"\nsequences within  it?"
' '
"single line string contained in multiple physical liens"
L"string"
U"string"
u"string"
u8"string"
"an unterminated string"
' '
' '
' '
'u'
'\\'
